test/org/sagebionetworks/bridge/services/AuthenticationServiceMockTest.java

```diff
@@ -264,7 +265,7 @@ public void adminSignInWithPhone() {
                 .copyOf(PARTICIPANT).withRoles(Sets.newHashSet(Roles.RESEARCHER)).build();
         doReturn(account).when(accountDao).authenticate(study, PHONE_PASSWORD_SIGN_IN);
         doReturn(participant).when(participantService).getParticipant(study, account, false);
-        doReturn(UNCONSENTED_STATUS_MAP).when(consentService).getConsentStatuses(any());
+        doReturn(UNCONSENTED_STATUS_MAP).when(consentService).getConsentStatuses(any(), any());
```

Comment:
reauthTokenRequired() should actually be reauthThrowsUnconsentedException().

Reply:
Updated with the unconsented status map and a catch clause to verify it is there and is the cause of the exception being thrown.